src/main/java/uk/gov/digital/ho/pttg/application/namematching/candidates/AbbreviatedNamesFunctions.java

```diff
@@ -13,24 +13,34 @@
 
     private static final String ANY_LETTER_INCLUDING_UNICODE_MATCHER = "\\p{L}\\p{M}*+";
     private static final String FULL_STOP_SPACE_MATCHER = "\\.\\s+";
+    private static final String APOSTROPHE_SPACE_MATCHER = "\'\\s+";
+
     private static final String FULL_STOP_SPACE_BETWEEN_NAMES_PATTERN = ANY_LETTER_INCLUDING_UNICODE_MATCHER + FULL_STOP_SPACE_MATCHER + ANY_LETTER_INCLUDING_UNICODE_MATCHER;
+    private static final String APOSTROPHE_SPACE_BETWEEN_NAMES_PATTERN = ANY_LETTER_INCLUDING_UNICODE_MATCHER + APOSTROPHE_SPACE_MATCHER + ANY_LETTER_INCLUDING_UNICODE_MATCHER;
+
     private static final Pattern FULL_STOP_SPACE_PATTERN_REGEX = Pattern.compile(FULL_STOP_SPACE_BETWEEN_NAMES_PATTERN);
+    private static final Pattern APOSTROPHE_SPACE_PATTERN_REGEX = Pattern.compile(APOSTROPHE_SPACE_BETWEEN_NAMES_PATTERN);
 
-    private static final String FULL_STOP_SPACE_NEGATIVE_LOOK_BEHIND = "(?<!(\\.|\\s))";
-    private static final String SPACE_NOT_PRECEDED_BY_FULL_STOP_OR_SPACE_PATTERN = FULL_STOP_SPACE_NEGATIVE_LOOK_BEHIND + "\\s+";
+    private static final String ABBREVIATION_SPACE_NEGATIVE_LOOK_BEHIND = "(?<!(\\.|'|\\s))";
+    private static final String SPACE_NOT_PRECEDED_BY_ABBREVIATION_OR_SPACE_PATTERN = ABBREVIATION_SPACE_NEGATIVE_LOOK_BEHIND + "\\s+";
 
     static boolean doesNotContainAbbreviatedNames(InputNames inputNames) {
-        if (nameContainsFullStopSpaceBetweenNames(inputNames.fullFirstName())) {
+        if (nameDoesContainAbbreviationSpaceBetweenNames(inputNames.fullFirstName())) {
```

Comment:
I don't think the introduction of Does has helped the name at all. nameContainsAbbreviationSpaceBetweenNames seems fine to me and makes the code read more like an English sentence.

Reply:
Resolved.